proposals.go

```diff
@@ -0,0 +1,93 @@
+package dcrlibwallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+const (
+	proposalsEndpoint = "https://proposals.decred.org/api/v1/proposals/vetted"
+	policyEndpoint    = "https://proposals.decred.org/api/v1/policy"
+)
+
+func (mw *MultiWallet) createHTTPClient() {
+	mw.httpClient = &http.Client{
+		Transport: &http.Transport{
+			MaxIdleConnsPerHost: 2,
+		},
+		Timeout: time.Duration(10) * time.Second,
+	}
+}
+
+func (mw *MultiWallet) get(url string, responseData interface{}) error {
+	res, err := mw.httpClient.Get(url)
+	if err != nil {
+		return err
+	}
+	defer res.Body.Close()
+
+	return json.NewDecoder(res.Body).Decode(responseData)
+}
+
+func (mw *MultiWallet) getPolicy() (*Policy, error) {
```

Comment:
Make the function names more descriptive. For example, this function could be `serverPolicy()`.